ingest-app/src/cmr/ingest/services/granule_bulk_update/opendap/opendap_util.clj

```diff
@@ -31,27 +31,28 @@
   https://opendap.*.earthdata.nasa.gov/* and no more than one that does not match the pattern.
   Returns the parsed urls in a map under keys :cloud and :on-prem with the corresponding urls."
   [url]
```

Comment:
The difference checker went wild here - all that changed is that the function body is now wrapped in a `(when url ...)`